doc/developers/maintainer.rst

```diff
@@ -17,25 +27,54 @@ Before a release
    change log is reasonably well curated. Some tools for these tasks include:
 
    - ``maint_tools/sort_whats_new.py`` can put what's new entries into
-     sections.
+     sections. It's not perfect, and requires manual checking of the changes.
+     If the whats new list if well curated, it may not be necessary.
 
    - The ``maint_tools/whats_missing.sh`` script may be used to identify pull
      requests that were merged but likely missing from What's New.
 
-Preparing a bug-fix-release
-...........................
+Permissions
+...........
+
+The release manager requires a set of permissions on top of the usual
+permissions given to maintainers, which includes:
+
+- *maintainer* role on ``scikit-learn`` projects on ``pypi.org`` and
+  ``test.pypi.org``, separately.
+- become a member of the *scikit-learn* team on conda-forge.
+- *maintainer* on ``https://github.com/MacPython/scikit-learn-wheels``
 
-Since any commits to a released branch (e.g. 0.999.X) will automatically update
-the web site documentation, it is best to develop a bug-fix release with a pull
+Preparing a release PR
+......................
+
+Since any commits to an existing branch (e.g. 0.999.X) will automatically
+update the web site documentation, it is best to develop a release with a pull
 request in which 0.999.X is the base. It also allows you to keep track of any
```

Comment:
Not sure I understand what "0.999.X is the base" means.  does it mean that the PR tries to merge things into the 0.999.X branch? Is there any other way to do it anyway?

Reply:
For me the two git commands a few lines later explain what it is. I also didn't understand what it meant from the text.